src/main/resources/org/jenkinsci/plugins/JiraTestResultReporter/JiraTestDataPublisher/config.jelly

```diff
@@ -105,6 +116,33 @@
             else
                 which.style.display="block"
         }
+        
+        
+        document.onreadystatechange = function(){
+           selectTag = document.getElementById('selectNoOfBugs');
+           if(selectTag) {
+		       numOfBugs = selectTag.value;
+		       	if (document.readyState === "complete") { 
+					for(i=0;i&lt;=15;i++) {
```

Comment:
I am assuming this is to populate the drop down with 15 elements/values? In the future will 15 be too limiting? Should we just make this a text field and allow the user to put whatever value they wish?